mesheryctl/internal/cli/root/version.go

```diff
@@ -130,7 +146,7 @@ func checkMesheryctlClientVersion(build string) {
 	}
 	// If user is running an outdated release, let them know.
 	if res.Outdated {
-		logrus.Info("\n  ", build, " is not the latest release. Update to v", res.Current, ".")
+		logrus.Info("\n  ", build, " is not the latest release. Update to ", res.Current, ".")
```

Comment:
@metonymic-smokey  mesheryctl was printing `vv0.x.x` before?

Reply:
Yep @piyushsingariya 
![image](https://user-images.githubusercontent.com/48997495/125247762-eb347e80-e310-11eb-949c-4c6503e77215.png)

Reply:
Just realised this today